edgelet/iotedge/test-files/config/ca-certs-est/super-config.toml

```diff
@@ -21,6 +21,8 @@ value = "YXppb3QtaWRlbnRpdHktc2VydmljZXxhemlvdC1pZGU="
 
 [preloaded_certs]
 
+[tpm]
```

Comment:
Is it necessary to add this? AFAIK, TPM is only used for DPS+TPM provisioning, so isn't relevant to other scenarios. Can we avoid having this in the config when it's not needed?

Reply:
The unit tests fail without this change, but we can look into a better solution. Ideally whatever we do should be consistent with the serialization of the tpm config in I-I-S.

@onalante-msft - Any thoughts on this? I believe you added the [tpm] section in a recent I-I-S PR.

Reply:
@gordonwang0 - I think removing this will require changes to the config structs and/or serialization code in iot-identity-service, which I think should be handled in a separate set of PRs (one for iot-identity-service followed by one for iotedge).

Reply:
Does this break backwards compatibility in any way? If a the super config does not have the [tpm] section, will I have to add an empty [tpm] section so that `iotedge config apply` works as before?

Reply:
[The `[tpm]` section is `#[serde(default)]`,](https://github.com/Azure/iot-identity-service/blob/469396c820f55bb5bcfab28088f33b931448a9fd/aziotctl/aziotctl-common/src/config/super_config.rs#L70-L71) so not having it will continue to be fine.

It's just missing the `#[serde(skip_serializing_if = "...::is_default")]` which is why it gets serialized by default.

Reply:
Should it have the `serde(skip_serializing_if = ..)` attribute? I initially opted not to add it since the fields above were also without the attribute.